subsys/mgmt/hawkbit/hawkbit.c

```diff
@@ -71,6 +75,7 @@ struct hawkbit_download {
 	int download_progress;
 	size_t downloaded_size;
 	size_t http_content_size;
+	size_t firmware_size;
```

Comment:
Who initializes this newly introduced member? I see it being used in the code, but I don't see it being set to some value at any place. This is specifically concerning given that the value is used as divisor at line 990.

Reply:
It's added now. It's set on line 520 now.